src/CondFormatManager.cpp

```diff
@@ -64,48 +100,78 @@ void CondFormatManager::removeItem()
     delete item;
 }
 
-void CondFormatManager::upItem()
+void CondFormatManager::moveItem(int offset)
 {
     if (!ui->tableCondFormats->currentIndex().isValid())
         return;
 
     int selectedRow = ui->tableCondFormats->currentIndex().row();
-    if(selectedRow == 0)
+    int newRow = selectedRow + offset;
+    if(newRow < 0 || newRow >= ui->tableCondFormats->topLevelItemCount())
         return;
 
-    QTreeWidgetItem* item;
+    QTreeWidgetItem* item = ui->tableCondFormats->topLevelItem(selectedRow);
+
+    // Rescue widgets, since they will be deleted, and add them later.
+    QFontComboBox* fontCombo = qobject_cast<QFontComboBox*>(ui->tableCondFormats->itemWidget(item, ColumnFont));
```

Comment:
The font combobox is too wide on my system and overlaps the other controls. Either we need to make sure it's not as wide or somehow fix the automatic column resizing. Or maybe this problem even depends on the Qt version?
![Screenshot_20190928_153111](https://user-images.githubusercontent.com/3153504/65817227-0817c480-e205-11e9-9369-be92a36ac3a0.png)

Reply:
That's weird! This is how it looks in my version:
![imagen](https://user-images.githubusercontent.com/13812910/65818742-e9212e80-e214-11e9-8f90-06ab82f94f6a.png)

Setting a maximum width could make the trick, but it looks like a Qt bug. I have another problem when a row is moved to the top:

![imagen](https://user-images.githubusercontent.com/13812910/65818803-71073880-e215-11e9-850e-bfb012dd369b.png)

A simple resize of the window fixes the layout.